deep_contract_auth/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
```

Comment:
The Makefile for this new example is missing.

Reply:
It's kind of trivial as this example is not intended to be built as Wasm (that would require 3 separate contracts, which is way beyond the point of what this demonstrates). But added for consistency.

Reply:
Is that why the Makefile uses `cargo build` instead of `soroban contract build`?

Reply:
Yeah, I don't want to build wasm for this one (I'm not even sure what would happen).